.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,10 @@
 ## Checklist
 _put `n/a` if item is not relevant to PR changes_
 
-- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
+- [ ] Has the CHANGELOG been updated?
+- [ ] Has the README been updated?
+- [ ] Has the RELEASE_GUIDELINES been updated?
+- [ ] Has the MIGRATION been updated in regards to the changes included in the PR?
 - [ ] Have new automated tests been implemented?
 - [ ] Have new manual tests been written down?
```

Comment:
I've noticed that especially with screen updates/new screens and upcoming copy updates, we're also likely to change existing tests rather than just add new ones. Would be worth rephrasing these 2 checklist points while we're at it now.